commcare-cloud-bootstrap/bootstrap.sh

```diff
@@ -15,3 +15,8 @@ commcare-cloud $ENV bootstrap-users --quiet --branch=$BRANCH
 commcare-cloud $ENV deploy-stack --skip-check --quiet -e 'CCHQ_IS_FRESH_INSTALL=1' --branch=$BRANCH
 
 commcare-cloud $ENV fab deploy:confirm=no,skip_record=yes --show=debug --set ignore_kafka_checkpoint_warning=true --branch=$BRANCH
+
+commcare-cloud $ENV django-manage check_services
+
+proxy=$(grep -A1 '\[pvtest-proxy-0\]' environments/pvtest/inventory.ini | tail -n 1| awk '{print $2}' | awk -F'=' '{print $2}')
```

Comment:
this might be intentional, but did you mean to leave `pvtest` hardcoded in there?

Reply:
That was a mistake, fixed here: https://github.com/dimagi/commcare-cloud/pull/2004/commits/60ddfa12ec01b235616b64be7c3ec53f36425ed2